src/layouts/components/Editor/LinkBubbleMenu.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  useDisclosure,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  InputGroup,
+  InputLeftAddon,
+} from "@chakra-ui/react"
+import { Button, Input } from "@opengovsg/design-system-react"
+import { BubbleMenu } from "@tiptap/react"
+import { useState } from "react"
+
+import { useEditorContext } from "contexts/EditorContext"
+import { useEditorModal } from "contexts/EditorModalContext"
+
+const LinkButton = () => {
+  const { editor } = useEditorContext()
+  const { onClose, onOpen, isOpen } = useDisclosure()
+  const { showModal } = useEditorModal()
+  const [href, setHref] = useState("")
+
+  return (
+    <>
+      <button
+        // TODO: Replace with a nice looking button
+        style={{
+          border: "1px solid black",
+          borderRadius: "5px",
+          padding: "1px 6px",
+          backgroundColor: "white",
+        }}
+        type="button"
+        onClick={() => {
+          // NOTE: If the link is an absolute link,
+          // it's not a file on Isomer so we will just allow them to change the link
+          // using the link modal.
+          if (
+            (editor.getAttributes("link").href as
+              | string
+              | undefined)?.startsWith("http")
+          ) {
+            onOpen()
+          } else {
+            // Otherwise, show the file modal
+            // and let the user select a file to link to.
+            showModal("files")
+          }
+        }}
+      >
+        Change link
+      </button>
+      <Modal isOpen={isOpen} onClose={onClose} size="sm">
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader>Update link</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <InputGroup size="sm">
+              <InputLeftAddon>https://</InputLeftAddon>
```

Comment:
I think similar comment as below, this assumption might be too strong to make that:
1. Users only add external links - they can add relative ones too
2. Users always add links to https - some sites might not have https, or the user might be linking to an email (i.e `mailto:test@example.com`)

Reply:
sure, i can remove this! actually this opens up another can of worms (users linking to insecure sites) but i think we're better off keeping the existing behaviour for now.